packages/client/src/test.cairo

```diff
@@ -39,25 +37,21 @@ struct UtreexoArgs {
 /// Receives arguments in a serialized format (Cairo serde).
 /// Panics in case of a validation error or chain state mismatch.
 /// Prints result to the stdout.
-fn main(arguments: Array<felt252>) -> Array<felt252> {
+#[executable]
+fn main(args: Args) {
     println!("Running integration test... ");
-    let mut args = arguments.span();
-
-    let Args {
-        mut chain_state, blocks, expected_chain_state, utreexo_args, execute_script,
-    } = Serde::deserialize(ref args).expect('Failed to deserialize');
-
+    let Args { mut chain_state, blocks, expected_chain_state, utreexo_args } = args;
     let mut utxo_set: UtxoSet = Default::default();
 
     for block in blocks {
-        match chain_state.validate_and_apply(block, ref utxo_set, execute_script) {
+        match chain_state.validate_and_apply(block, ref utxo_set) {
```

Comment:
why `execute_script` is removed?

Reply:
Added `make build-with-shinigami/build` commands to build executable with/without script execution enabled.